dot/telemetry/afg_authority_set.go

```diff
@@ -3,25 +3,25 @@
 
 package telemetry
 
-// afgAuthoritySetTM is a telemetry message of type `afg.authority_set` which is
+// AfgAuthoritySetTM is a telemetry message of type `afg.authority_set` which is
 // meant to be sent when authority set changes (generally when a round is initiated)
-type afgAuthoritySetTM struct {
+type AfgAuthoritySetTM struct {
 	AuthorityID    string `json:"authority_id"`
 	AuthoritySetID string `json:"authority_set_id"`
 	// Substrate creates an array of string of authority IDs. It JSON-serialises
 	// that array and send that as a string.
 	Authorities string `json:"authorities"`
 }
 
-// NewAfgAuthoritySetTM creates a new afgAuthoritySetTM struct.
-func NewAfgAuthoritySetTM(authorityID, authoritySetID, authorities string) Message {
-	return &afgAuthoritySetTM{
+// NewAfgAuthoritySetTM creates a new AfgAuthoritySetTM struct.
+func NewAfgAuthoritySetTM(authorityID, authoritySetID, authorities string) AfgAuthoritySetTM {
+	return AfgAuthoritySetTM{
```

Comment:
Return this as a pointer so it can directly be received as a `Message` interface (since an interface must be a pointer)

Reply:
Changed to return a pointer!